server/repository/src/db_diesel/filter_sort_pagination.rs

```diff
@@ -22,6 +22,103 @@ impl SimpleStringFilter {
     }
 }
 
+#[derive(Clone, PartialEq, Debug)]
+pub struct StringFilter {
```

Comment:
I think at this point, when we have three filters, it's easier on devs to consolidate them all into one. Not in this PR, but would be good to keep in mind

Reply:
Yes at least StringFilter and SimpleStringFilter